tests/StorageBackendTest.php

```diff
@@ -115,6 +115,31 @@ public function storageBackendOptionsProvider(): \Generator
         ];
     }
 
+    /**
+     * @dataProvider storageBackendOptionsProvider
+     */
+    public function testUpdateStorageBackend(array $options)
+    {
+        $backend = $this->client->createStorageBackend($options);
+
+        $wrongOptions = [
+            'password' => 'invalid',
+        ];
+
+        try {
+            $this->client->updateStorageBackend($backend['id'], $wrongOptions);
+            $this->fail('Should fail!');
+        } catch (ClientException $e) {
+            $this->assertSame('Failed to connect using the supplied credentials', $e->getMessage());
+        }
+
+        $options = [
+            'password' => getenv('KBC_TEST_SNOWFLAKE_BACKEND_PASSWORD'),
+        ];
+
+        $this->client->updateStorageBackend($backend['id'], $options);
```

Comment:
tady bych tomu aspon neco malo assertnul

Reply:
dal som assert aspon na to co ten update vrati, nic ine tam asi nedokazem skontrolovat. :/